src/containers/profile/Profile.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react";
+import GithubProfileCard from "../../components/githubProfileCard/GithubProfileCard";
+import axios from "axios";
+import { openSource } from "../../portfolio";
+import { Fade } from "react-reveal";
+import Contact from "../contact/Contact";
+
+
+export default function Profile() {
+  const [prof, setrepo] = useState([]);
+
+  function setProfileFunction(array) {
+    setrepo(array);
+  }
+  function getProfileData() {
+      const client=axios.get("https://api.github.com/users/"+openSource.githubUserName)
+            .then(function(response){
+                    setProfileFunction(response.data);
+      });
+  }
+  useEffect(() => {
+    getProfileData();
+  }, []);
+  if (openSource.showProfile === "true"){
+  return (
+    <Fade bottom duration={1000} distance="20px">
```

Comment:
Can we move all this code outside ``GithubProfileCard`` to inside that component?

It would boost readability and make the code easier to understand.

Remove react-reveal too after doing this.